source/dub/generators/build.d

```diff
@@ -387,6 +414,14 @@ class BuildGenerator : ProjectGenerator {
 			const pdbFilename = filenames[0].setExtension(".pdb");
 			if (existsFile(build_path ~ pdbFilename))
 				filenames ~= pdbFilename;
+
+			const importLibraryLocation = filenames[0].setExtension(".lib");
+			if (existsFile(build_path ~ importLibraryLocation))
+				filenames ~= importLibraryLocation;
+
+			const exportFilesLocation = filenames[0].setExtension(".exp");
+			if (existsFile(build_path ~ exportFilesLocation))
+				filenames ~= exportFilesLocation;
```

Comment:
I think this entire conditional block can be removed now that we have the dependencyBinariesToCopy parameter here?